DNN Platform/Website/Default.aspx.cs

```diff
@@ -645,6 +653,110 @@ private void InitializePage()
             }
         }
 
+        private string GenerateCssCustomProperties()
+        {
+            string cacheKey = $"Dnn_Css_Custom_Properties_{this.PortalSettings.PortalId}";

```

Comment:
Does this cache key get set? Should this be using `CBO.GetCachedObject`?

Reply:
I kinda ran out of time last week on this, it will get used soon... I actually probably have to move that in some other place as the module to manage it will need to clear that cache.

Reply:
Ok, we will need that key in the module too and I don't want to module and DNN to both have it (don't repeat yourself).... I think this is a nice place to reuse it ?

Reply:
@bdukes any thoughts ?

Reply:
I would just add it to the collection of keys in `DataCache`
https://github.com/dnnsoftware/Dnn.Platform/blob/develop/DNN%20Platform/Library/Common/Utilities/DataCache.cs#L43-L74

Reply:
Oh, good point, done in https://github.com/dnnsoftware/Dnn.Platform/pull/5851/commits/4f3966f44bfb3c5140a290415d5430cfdb449c84